src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/DefaultProjectConfigurationFilePathStoreTest.cs

```diff
@@ -10,13 +10,35 @@ namespace Microsoft.VisualStudio.LanguageServerClient.Razor
 {
     public class DefaultProjectConfigurationFilePathStoreTest
     {
+        [Fact]
+        public void Set_ResolvesRelativePaths()
+        {
+            // Arrange
+            var store = new DefaultProjectConfigurationFilePathStore();
+            var projectFilePath = @"C:\project.csproj";
+            var configurationFilePath = @"C:\project\subpath\..\obj\project.razor.json";
```

Comment:
What happens if `var configurationFilePath = @".\..\obj\project.razor.json";`? That's what I assumed was meant by "ResolvesRelativePaths".

Reply:
@ryanbrandenburg it'll take your `Directory.GetCurrentDirectory` (equates to where the application was launched from) and then it'll go up one directory from that 😄

Reply:
I guess what I really meant was it might be a good idea to test this against a non-rooted path if that's possible in this case.

Reply:
Ah, technically could; however, the API wouldn't return a testable output given it uses the current directory which will be influenced by what machine boots the test process